app/styles/ui/_cloneable-repository-filter-list.scss

```diff
@@ -16,4 +16,20 @@
       color: var(--text-secondary-color);
     }
   }
+
+  .clone-repository-list-item {
+    .name {
+      flex-grow: 1;
+    }
+
+    .archived {
+      flex-shrink: 0;
+      margin-left: var(--spacing-half);
+      font-size: var(--font-size-xs);
+      border: var(--base-border);
```

Comment:
The border color doesn't look good in dark mode IMO, I would expect a lighter color:

![image](https://github.com/desktop/desktop/assets/1083228/12e0a09d-8419-491d-b106-ae7fd2761f48)